### PR TITLE
fix runtime image test failures for node

### PR DESCRIPTION
**Pull request by arroyc**
fixing some node runtime image test failures

- [X] The purpose of this PR is explained in this message or in an issue. If an issue please include a reference as \#\<issue\_number\>.
- [X] Tests are included and/or updated for code changes.
- [X] Proper license headers are included in each file.

### CODE REVIEW

**Comment by arroyc** [comment]
image

**Comment by qianz2** [comment]
Thanks, I also found out this test not working locally. Still curious why tests ran from build agent failed. Let's try again from build agent..

**Review by qianz2 — APPROVED** [review]

File: images/runtime/node/installDependencies.sh [hunk 1]
```diff
@@ -30,7 +30,7 @@ echo "Version of npm: $currentNpmVersion"
 # Upgrade npm to the latest available version
 if [[ $nodeVersionMajor -ge 10  ]]; then
     echo "Upgrading npm..."
-    npm install npm -g -loglevel silent
```
Comment (arroyc) [inline images/runtime/node/installDependencies.sh:33]: @purva-vasudeo this is where the problem occurred

**Comment by qianz2** [comment]
Please make sure the nightly build also passed here

**Comment by arroyc** [comment]
> Thanks, I also found out this test not working locally. Still curious why tests ran from build agent failed. Let's try again from build agent..



> Please make sure the nightly build also passed here

I'm not going to run the nightly build for this PR .... your  .net 6 runtime PR have integreation tests .. so you can run it before merging your Pr.

**Pull request merged**